src/vs/platform/environment/node/environmentService.ts

```diff
@@ -114,6 +114,9 @@ export class EnvironmentService implements IEnvironmentService {
 	@memoize
 	get sharedIPCHandle(): string { return `${getIPCHandlePrefix()}-${pkg.version}-shared${getIPCHandleSuffix()}`; }
 
+	@memoize
+	get nodeCachedDataDir(): string { return path.join(this.userDataPath, 'CachedData', pkg.version); }
```

Comment:
@jrieken ok but that basically means each insider user now has the performance penalty to pay to get the cached data and write to disk instead of also having the performance gain on insiders?